tests/integration-tests/tests/storage/storage_common.py

```diff
@@ -161,18 +161,24 @@ def _write_user_data(efs_id, random_file_name):
         """  # noqa: E501
 
 
-def test_efs_correctly_mounted(remote_command_executor, mount_dir):
-    logging.info("Testing ebs {0} is correctly mounted".format(mount_dir))
-    result = remote_command_executor.run_remote_command("df | grep '{0}'".format(mount_dir))
+def test_efs_correctly_mounted(remote_command_executor, mount_dir, tls=False):
```

Comment:
Why having a parameter for in-transit encryption but not for IAM authentication?

Reply:
Because this integration test will be improved by another PR. In other integration test PR will be created after the support on cluster configuration file parameter is done is done

Reply:
I'm not getting why we are currently blocked to add the IAM part but we are not for the TLS.
May you please clarify this point?

Reply:
I see from the TODO in PR description that you're going to improve the test once the actual changes in cookbook will be in place.
Nevertheless, since both TLS and IAM are actually not in place, why adding TLS and not IAM?

Reply:
It's not a concern, it's because I'm not getting why we want one and not the other

Reply:
Clarified offline. That's fine. It' sjuts because we already have the logic to check for TLS but we don't for IAM